websites/T/TikTok/dist/metadata.json

```diff
@@ -21,16 +21,12 @@
 		"vi_VN": "TikTok - xu hướng bắt đầu tại đây. Trên thiết bị hay trên web, người dùng có thể xem và khám phá hàng triệu các video ngắn được cá nhân hoá."
 	},
 	"url": "www.tiktok.com",
-	"version": "2.2.7",
+	"version": "2.3.0",
 	"logo": "https://i.imgur.com/KBAjCOX.png",
 	"thumbnail": "https://i.imgur.com/QAxUXB4.png",
 	"color": "#000000",
 	"category": "socials",
-	"tags": [
-		"tiktok",
-		"video",
-		"social"
-	],
+	"tags": ["tiktok", "video", "social"],
```

Comment:
Revert this change

Reply:
Prettier auto formats it like this?

Reply:
Run SE instead. SE should have the preferred format on the repo; you wouldn’t have needed to make the change if it was preferred